src/input/event/value.rs

```diff
@@ -246,6 +251,52 @@ impl InputValue {
                     Gamepad::Accelerometer => Err(TranslationError::NotImplemented),
                     // Gyro -> ...
                     Gamepad::Gyro => Err(TranslationError::NotImplemented),
+                    Gamepad::Dial(_) => match target_cap {
+                        // Gamepad Button -> None
```

Comment:
These comments should all be Gamepad Dial ->

Reply:
Removed comments